src/run.ts

```diff
@@ -18,6 +18,7 @@ export async function run() {
    const action: Action | undefined = parseAction(
       core.getInput('action', {required: true})
    )
+   const annotations: {[key: string]: string} = {}
```

Comment:
you aren't passing in the annotations with core.getInput

Reply:
also keep in mind that with core.getInput you will be parsing it as a string. You will have to parse it as an object